### PR TITLE
Added: Healenium Dependency and Configration

### PR DESCRIPTION
**Pull request by heyrmi**
# Summary

## Compatibility

| Browser | Implemented |
|---------|:-----------:|
| Chrome  |     ✅❌      |
| Firefox |     ✅❌      |

## Checklist

- [ ] I have tested this on all browsers
- [ ] I added the documentation in `README.md`
- [ ] I updated function description

### CODE REVIEW

**Comment by dipjyotimetia** [comment]
Few things,
Can can you please update the gradle dependency referring to the previous once.
Also please add an example test scripts using the new tool.

**Comment by heyrmi** [comment]
Okay. I will be making a switch in a config file and manage the driver accordingly for Healenium.

File: infra/db/sql/init.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CREATE SCHEMA healenium AUTHORIZATION healenium_user;
+GRANT USAGE ON SCHEMA healenium TO healenium_user;
```
Comment (dipjyotimetia) [inline infra/db/sql/init.sql:2]: add a newline

File: src/test/resources/healenium.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+recovery-tries = 1
+score-cap = 0.5
+heal-enabled = true
+serverHost = localhost
+serverPort = 7878
+imitatePort = 8000
```
Comment (dipjyotimetia) [inline src/test/resources/healenium.properties:6]: add a new line

File: infra/db/docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+version: "3.3"
```
Comment (dipjyotimetia) [inline infra/db/docker-compose.yaml:1]: update to version "3.8"

File: infra/db/docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+services:
+
+  db:
+    image: postgres:11-alpine
```
Comment (dipjyotimetia) [inline infra/db/docker-compose.yaml:6]: alwyas try to use latest stable image:
`postgres:14-alpine`

File: infra/db/docker-compose.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    ports:
+      - "5432:5432"
+    volumes:
```
Comment (dipjyotimetia) [inline infra/db/docker-compose.yaml:9]: try to add a container_name as well so that its not pickup defaultname

File: build.gradle [hunk 6]
```diff
@@ -150,6 +150,7 @@ dependencies {
     implementation "org.testcontainers:gcloud:${containerVersion}"
     implementation 'org.projectlombok:lombok:1.18.24'
     annotationProcessor 'org.projectlombok:lombok:1.18.22'
+    compile group: 'com.epam.healenium', name: 'healenium-web', version: '3.2.4'
```
Comment (dipjyotimetia) [inline build.gradle:153]: implementation 'com.epam.healenium:healenium-web:3.2.4'

**Review by dipjyotimetia — CHANGES_REQUESTED** [review]
Along with these changes, change the folder structure from infra to healenium

File: infra/db/docker-compose.yaml (hunk 4 above)
Comment (heyrmi) [reply infra/db/docker-compose.yaml:6]: Will keep in mind. Thanks.

**Review by dipjyotimetia — APPROVED** [review]
lgtm

**Pull request merged**